BD-4518 Suppression List updates (#9434)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/segments/suppression_lists.md

## Suppression list usage

> Use **User Lookup** in campaigns and Canvases to check if suppression list membership prevented a user from receiving that campaign or Canvas. Applied suppression lists can also be located in the **Summary** step.

Reviewer: ```suggestion
To check if your suppression list prevented a user from receiving a message, use **User Lookup** in the **Target Audience** step within your campaign or Canvas. Here, you'll be able to see which suppression list they're part of.

!["User Lookup" window showing that a user is in a suppression list.][7]{: style="max-width:70%;"}

{% alert tip %}
You can also find applied suppression lists in the **Summary** step.
{% endalert %}
```

While creating a campaign or Canvas, use **User Lookup** within the **Target Audience** step to search for a user, and if they aren't in the target audience, you can see the suppression list they're part of. 

!["User Lookup" window showing that a user is in a suppression list.][7]{: style="max-width:70%;"}

> ### For campaigns

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
{% tabs local %}
{% tab campaign %}
```

If a user is in a suppression list, they won't receive a campaign for which that suppression list applies. Refer to [Messages not affected by suppression lists](#messages-not-affected-by-suppression-lists) for cases when a suppression list won't apply.

![The "Suppression Lists" section with one active suppression list, called "Low marketing health scores".][5]

> 

Reviewer: ```suggestion
{% endtab %}
```

> ### For Canvases

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
{% tab canvas %}
```

For example, let’s say you have a Canvas with an applied suppression list. The Canvas has a User Update step followed by a Message step. In this scenario, suppression list users will enter the Canvas, proceed through the User Update step (where the user may be updated, based on how that step is configured), and then exit at the Message step (at which point the user will be included in the “Exited” metrics). 

> 

Reviewer: ```suggestion
{% endtab %}
{% endtabs %}
```
